projects/libs/core-library-typescript/src/state-storage/state-storage.ts

```diff
@@ -94,12 +97,30 @@ export class StateStorage {
     if ((this.lastBlockHeight == undefined) || (block.header.height == this.lastBlockHeight + 1)) {
       logger.debug("Processing block:", block.header.height);
 
-      for (const { contractAddress, key, value} of block.body.stateDiff) {
+      for (const { contractAddress, key, value } of block.body.stateDiff) {
         this.kvstore.set(contractAddress, key, value);
       }
       this.lastBlockHeight = block.header.height;
     } else {
-      throw new Error(`Unexpected block Height: ${block.header.height}. Expected ${ this.lastBlockHeight + 1 }, Out of sync?`);
+      throw new Error(`Unexpected block Height: ${block.header.height}. Expected ${this.lastBlockHeight + 1}, Out of sync?`);
     }
   }
+
+  public async startupCheck(): Promise<StartupStatus> {
+
+    if (!this.kvstore) {
+      return <StartupStatus>{ name: this.SERVICE_NAME, status: STARTUP_STATUS.FAIL, message: "Missing kvstore" };
+    }
+
+    if (!this.pollIntervalMs) {
```

Comment:
No need to check this. It's already initialized in the constructor.

Reply:
This is a defensive check - what if someone decides to remove this from ctor..